src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx

```diff
@@ -126,6 +126,15 @@ To instrument [`Transactions`](/docs/apm/applications-menu/monitoring/transactio
         [`NewRelic.ignoreApdex()`](http://newrelic.github.io/java-agent-api/javadoc/index.html?com/newrelic/api/agent/NewRelic.html)
       </td>
     </tr>
+    <tr>
+      <td>
+        To associate a user ID with the current transaction and any errors noticed during the transaction's lifespan by setting an `enduser.id` agent attribute.
```

Comment:
I find this a little hard to parse; hard to tell what clause applied to what clause; could you rework it a little bit?

Reply:
How does this sound:

Reply:
How does this sound?

Adds user tracking to the current transaction by setting the `enduser.id` attribute.